src/test/resources/emails/sessionClosingEmailForStudent.html

```diff
@@ -9,7 +9,7 @@
 <br>&nbsp;&nbsp; Feedback Session Name: First feedback session
 <br>&nbsp;&nbsp; Deadline: Fri, 30 Apr 2027, 11:59 PM
 <br>
-<strong>To submit feedback for the above session, please go to this Web address: </strong><a href="${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student5InCourse1%40gmail.tmt">${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student5InCourse1%40gmail.tmt</a>
+<strong>To submit feedback for the above session, please go to this Web address: </strong><a href="${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student5InCourse1%40gmail.tmt">${app.url}/page/studentFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session&key=${regkey.enc}&studentemail=student5InCourse1%40gmail.tmt</a>.
```

Comment:
Yes, let's do it in this PR. The extra code for supporting different versions of that message is unnecessary.